src/main/resources/redisson.yaml

```
@@ -0,0 +1,13 @@
+replicatedServersConfig:
+  password: ${REDIS_PASSWORD}
+  subscriptionConnectionMinimumIdleSize: ${REDIS_SLAVE_CONN_MIN}
+  subscriptionConnectionPoolSize: ${REDIS_SLAVE_CONN_MAX}
+  slaveConnectionMinimumIdleSize: ${REDIS_SLAVE_CONN_MIN}
+  slaveConnectionPoolSize: ${REDIS_SLAVE_CONN_MAX}
+  masterConnectionMinimumIdleSize: ${REDIS_MASTER_CONN_MIN}
+  masterConnectionPoolSize: ${REDIS_MASTER_CONN_MAX}
+  sslEnableEndpointIdentification: ${REDIS_SSL_ENABLED}
+  readMode: "MASTER"
+  subscriptionMode: "MASTER"
+  nodeAddresses:
+    - "${REDIS_PROTOCOL}://${REDIS_HOST}:${REDIS_PORT}"
```

```suggestion
    - "${REDIS_PROTOCOL:rediss}://${REDIS_HOST}:${REDIS_PORT}"
```

Maybe here we can default to rediss(ecure) protocol by default, overriding this value only for local environment, what do you think? (you should test that default value expansion work correctly here)
This way we can remove REDIS_PROTOCOL variable from all helm value